src/app/home/landing/landing.component.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../shared/scss/_selected_theme_variables.scss";
+:host {
+    /deep/ .section-title {
+        color: $black;
+        text-transform: capitalize;
+        font-weight: 600;
+    }
+    /deep/ .scrollbar-hidden {
+      overflow: hidden !important;
+    }
+    /deep/ .slider-section {
+      position: relative;
+      /deep/ .slider-control {
```

Comment:
No need to add `/deep/` here if parent `/deep/ .scrollbar-hidden` already has it